lib/Mojo/Base.pm

```diff
@@ -76,9 +76,8 @@ sub import {
   }
 
   # Module
-  elsif ((my $file = $flags[0]) && !$flags[0]->can('new')) {
-    $file =~ s!::|'!/!g;
-    require "$file.pm";
+  elsif ($flags[0] && !$flags[0]->can('new')) {
+    require(Mojo::Util::class_to_path($flags[0]));
```

Comment:
Parentheses are needed for Mojo::Util::class_to_path to avoid circular require issues. Parentheses are needed for require to avoid weird syntax errors possibly related to indirect object syntax.